fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java

```diff
@@ -195,6 +197,31 @@
         tripleFormat.setTimeZone(getTimeZone("GMT"));
     }
 
+    private static File TEST_FILE;
```

Comment:
If it is a question of triggering binary storage, you don't need anything _like_ a 50MB to do that. Refer to the [Modeshape configuration](https://github.com/fcrepo4/fcrepo4/blob/master/fcrepo-http-api/src/test/resources/test_repository.json#L14) used for this test and you will see that the minimum size to trigger binary storage is only 40 bytes.